inspector/src/components/graph/canvasGraphService.ts

```diff
@@ -38,17 +39,222 @@ export class CanvasGraphService {
         // First we clear the canvas so we can draw our data!
         this.clear();
 
-        // TODO: Draw each dataset
-        this.datasets.forEach((dataset: IPerfDataset) => {
-            // do drawing
+        // Get global min max of time axis (across all datasets).
+        let globalTimeMinMax = {min: Infinity, max: 0};

```

Comment:
shouldn't that be flipped? max is infinity and min is 0?

Reply:
I don't think so because if i set max to infinity and min to 0 when i do Math.min(globalTimeMinMax.min, timeMinMax.min); then the result will always be 0 and similarly for max where the result will always be infinity!